set.mm

```diff
@@ -534405,10 +534370,7 @@ sides of the biconditional correlate (they are the same), if they exist at
     $( Rederive ~ df-clab from ~ wl-clabv .  (Contributed by Wolf Lammen,
        31-May-2023.) $)
     wl-dfclab $p |- ( x e. { y | ph } <-> [ x / y ] ph ) $=
```

Comment:
@wlammen This theorem can be removed!?

Reply:
Probably not; the old proof should be restored with a proof discouragement tag.